docs/sources/gateway.md

```diff
@@ -104,15 +104,74 @@ Iterates over all listeners for the parent's `parentRef.sectionName`:
 
 ## Targets
 
-The targets of the DNS entries created from a \*Route are sourced from the following places:
+Targets are derived from a combination of Route annotations, Gateway annotations, and
+the Gateway's `status.addresses`. How these sources are combined is controlled (per Route)
+by the optional strategy annotation:
 
-1. If a matching parent Gateway has an `external-dns.alpha.kubernetes.io/target` annotation, uses
-   the values from that.
+```yaml
+external-dns.alpha.kubernetes.io/target-strategy: <strategy>
+```
+
+Supported strategies (current behavior for Gateway *Route sources):
+
+| Strategy         | Behavior | Fallback Order | Publishes Multiple Target Sets? |
+|------------------|----------|----------------|----------------------------------|
+| `route-preferred` (default when omitted or unrecognized) | Use Route targets if present; otherwise Gateway targets if present; otherwise addresses | Route → Gateway → Addresses | No (at most one set) |
```

Comment:
```suggestion
| `route-preferred` (default) | Use Route targets if present; otherwise Gateway targets if present; otherwise addresses | Route → Gateway → Addresses | No (at most one set) |
```